internal/cephfs/clone.go

```diff
@@ -158,7 +163,7 @@ func createCloneFromSnapshot(ctx context.Context, parentVolOpt, volOptions *volu
 	}
 	defer func() {
 		if err != nil {
-			if !errors.Is(err, ErrCloneInProgress) {
+			if !errors.Is(err, ErrCloneInProgress) || !errors.Is(err, ErrClonePending) {
```

Comment:
This check can be don in a helper function, something like `isRetryableCloneErr()`, making it all a little better maintainable.